docs/source/model_design/general_research.rst

```diff
@@ -679,11 +679,131 @@ Conducting Systematic Literature Searches/Reviews
 Directed Acyclic Graphs (Causal Diagrams)
 ------------------------------------------
 
-.. todo::
+.. important::
+
+  The figures represented in this section represent directed acyclic graphs/causal diagrams. These figures are distinct from compartamental model diagrams (such as SIR models). For our purposes we will represent directed acyclic graphs as *circles* connected by arrows and compartamental model diagrams as *squares* connected by arrows.
+
+  Additionally, for the purposes of directed acyclic graphs, variables will be labeled with capital letters (A, B, C, etc.) and arrows will be labeled with lower case letters (a, b, c, etc.).
+
+Introduction to Directed Acyclic Graphs
++++++++++++++++++++++++++++++++++++++++
+
+Directed acyclic graphs (DAGs), also termed causal diagrams, are composed of variables that are linked together by directional arrows. Critically, the solid arrows in DAGs represent a causal relationship between an exposure (start of the arrow) and outcome (node of the arrow), representing the *directional* aspect of DAGs. Notably, a directional arrow in a DAG does not offer any information about whether the exposure causes an increased or decreased probability of the outcome, only that there is some causal realtionship bewteen the exposure and outcome. 
```

Comment:
I mostly like how this is written, but my one complaint is that DAGs are a general mathematical concept with many different applications, and the causal diagrams described here are a specific application of DAGs.  I can't think of a good way to reword this part right now, but these two Wikipedia pages provide some context, including related models and other applications of DAGs:

https://en.wikipedia.org/wiki/Directed_acyclic_graph
https://en.wikipedia.org/wiki/Causal_model#Causal_diagram

Reply:
Perhaps something like:

>[Directed acyclic graphs](https://en.wikipedia.org/wiki/Directed_acyclic_graph) (DAGs) are composed of nodes that are linked together by directional arrows, in such a way that the arrows do not form any [directed cycles](https://en.wikipedia.org/wiki/Cycle_graph#Directed_cycle_graph). For our purposes here, DAGS will represent [causal diagrams](https://ftp.cs.ucla.edu/pub/stat_ser/r332.pdf). Critically, the solid arrows in DAGs represent a causal relationship between an exposure variable (start of the arrow) and outcome variable (end of the arrow), representing the *directional* aspect of DAGs.

Note: I changed "node of the arrow" to "end of the arrow" - did you mean to write "node" (I'm not familiar with that usage)?

Reply:
A term-of-art for the directional arrows is "arcs", which we could also bring in, e.g. "that are linked together by directional arrows (arcs of the graph)"

Two alternative/additional links https://en.wikipedia.org/wiki/Directed_graph https://en.wikipedia.org/wiki/Glossary_of_graph_theory_terms